learning_resources/views.py

```diff
@@ -505,12 +532,15 @@ def learning_paths(self, request, *args, **kwargs):  # noqa: ARG002
                 relationship.position = index
                 relationship.save()
                 last_index = index
-            LearningResourceRelationship.objects.create(
-                parent_id=learning_path_id,
-                child_id=learning_resource_id,
-                relation_type=LearningResourceRelationTypes.LEARNING_PATH_ITEMS,
-                position=last_index + 1,
-            )
+
+            # Add new items as necessary
+            if learning_path_id not in list(current_parent_lists):
```

Comment:
I spend a while trying to debug why my conditional was not working, and it ended up being because (originally) `learning_path_id` was a string, whereas `current_parent_lists` was a list of integers.

That motivated me to make the two new serializers for validating request data.